index.js

```diff
@@ -67,3 +72,11 @@ async function postGraphilePlugin (fastify, opts) {
 module.exports = fastifyPlugin(postGraphilePlugin, {
   name: DECORATOR
 })
+
+module.exports.DEFAULT_INFLECTOR = DEFAULT_INFLECTOR
+
+function concatNestedArrays (objValue, srcValue) {
+  if (Array.isArray(objValue)) {
+    return objValue.concat(srcValue)
+  }
+}
```

Comment:
```suggestion
function concatNestedArrays (objValue, srcValue) {
  if (Array.isArray(objValue)) {
    return objValue.concat(srcValue)
  }
}

module.exports.DEFAULT_INFLECTOR = DEFAULT_INFLECTOR
module.exports = fastifyPlugin(postGraphilePlugin, {
  name: DECORATOR
})
```

I prefer these in the opposite order, with functions gathered in the middle and exports gathered at the bottom.